### PR TITLE
[8.0][ADD] social_media_pinterest addon

### PR DESCRIPTION
**Pull request by P4R**
This module adds social media link for Pinterest. By default, the icon will be shown on the footer, about us, and the Blog, like the original icons.

### CODE REVIEW

File: social_media_pinterest/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    'name': "Pinterest Social Media Icon Extension",
+    'summary': """Pinterest Extension for the social media icons from the
+    odoo core""",
+    'author': "Diagram Software, S.L."
+    "Odoo Community Association (OCA)",
+    'website': "http://www.diagram.es",
+    'license': 'AGPL-3',
+    'category': 'Social Media',
+    'version': '8.0.1.0.0',
+
```
Comment (andhit-r) [inline social_media_pinterest/__openerp__.py]: Remote empty lines

File: social_media_pinterest/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    'depends': [
+        'base',
+        'website',
+        'website_blog'
```
Comment (andhit-r) [inline social_media_pinterest/__openerp__.py]: I don't think it's wise to directly depend on website_blog. There are possibility that company want this but do not want the blog. You can create additional module to add this social media to blog.

File: social_media_pinterest/models/res_config.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+# © 2016 Diagram Software S.L.
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+from openerp.models import TransientModel
```
Comment (andhit-r) [inline social_media_pinterest/models/res_config.py]: from openerp import models

File: social_media_pinterest/models/res_config.py [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+from openerp import fields
+
+
+class WebsiteConfigSettings(TransientModel):
```
Comment (andhit-r) [inline social_media_pinterest/models/res_config.py]: models.TransientModel

File: social_media_pinterest/models/website.py [hunk 5]
```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+# © 2016 Diagram Software S.L.
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+from openerp.models import Model
```
Comment (andhit-r) [inline social_media_pinterest/models/website.py]: from openerp import models

Comment (andhit-r) [inline social_media_pinterest/models/website.py]: from openerp import models

File: social_media_pinterest/models/website.py [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+from openerp import fields
+
+
+class Website(Model):
```
Comment (andhit-r) [inline social_media_pinterest/models/website.py]: models.Model

**Review by andhit-r — CHANGES_REQUESTED** [review]
Hi @P4R , thanks for your contributions.

Functional Review: None
Code Review: You should adjust README.rst according latest OCA guidelines. Few comments inline

File: social_media_pinterest/README.rst [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+=============
+
+- Go in the backend under settings / website
+- You can now also add your Pinterest Account
+
+
+Usage
+=====
+
+- Normal social icons:
```
Comment (hbrunn) [inline social_media_pinterest/README.rst]: you can use two colons here to have the following indented paragraph(s) formatted as code

File: social_media_pinterest/__openerp__.py [hunk 8]
```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# © 2016 Diagram Software S.L.
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+{
+    'name': "Pinterest Social Media Icon Extension",
+    'summary': """Pinterest Extension for the social media icons from the
+    odoo core""",
+    'author': "Diagram Software, S.L."
```
Comment (hbrunn) [inline social_media_pinterest/__openerp__.py]: add a comma between this and oca. And better don't use commas in your own company name, as the app store splits the authors by comma

File: social_media_pinterest/__openerp__.py [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+    "Odoo Community Association (OCA)",
+    'website': "http://www.diagram.es",
+    'license': 'AGPL-3',
+    'category': 'Social Media',
+    'version': '8.0.1.0.0',
+    'depends': [
+        'base',
```
Comment (hbrunn) [inline social_media_pinterest/__openerp__.py]: you don't need this one

File: social_media_pinterest_blog/__openerp__.py [hunk 10]
```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# © 2016 Diagram Software S.L.
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+{
+    'name': "Pinterest Social Media Icon Extension for blog",
+    'summary': """Pinterest Extension for the social media icons from the
+    odoo core""",
+    'author': "Diagram Software, S.L."
```
Comment (hbrunn) [inline social_media_pinterest_blog/__openerp__.py]: same as above

File: social_media_pinterest_blog/README.rst [hunk 11]
```diff
@@ -0,0 +1,83 @@
...
+=============
+
+- Go in the backend under settings / website
+- You can now also add your Pinterest Account
+
+
+Usage
+=====
+
+- Normal social icons:
```
Comment (hbrunn) [inline social_media_pinterest_blog/README.rst]: same as above

**Review by hbrunn — CHANGES_REQUESTED** [review]
Did you check if you can recycle the pinterest icon legally? If so, also use a smaller version of it as icon and add the source to the credits section, if not, remove

**Comment by P4R** [comment]
@hbrunn Info about use of pinterest icon: https://business.pinterest.com/en/brand-guidelines

**Review by hbrunn — APPROVED** [review]

**Comment by hbrunn** [comment]
@andhit-r if you change your request to an approval, we can merge this

**Review by andhit-r — APPROVED** [review]

**Pull request merged**